EmerisHQ/demeris
# feat: remember last wallet and autologin

## Description

Branched off #1797 because it was almost impossible to test without that PR

Fixes: #1758 

## Feature flags

?VITE_FEATURE_USE_EMERIS_EXTENSION=1

## Testing

- Login using the EE wallet
- Refresh
- Check whether EE wallet still loads

---
